pycqed/analysis/process_tomography.py

```diff
@@ -1,10 +1,9 @@
 import numpy as np
 import qutip as qtp
 from pycqed.analysis import analysis_toolbox as a_tools
-from pycqed.analysis import composite_analysis as ca
 from matplotlib import pyplot as plt
 from mpl_toolkits.axes_grid1 import make_axes_locatable
-from plotting_tools import *
+from pycqed.analysis.tools.plotting import *
```

Comment:
I'm slightly confused now, is this the new structure? I think it looks great. But how does the analysis toolbox fit in this picture?

Reply:
I don't understand what you mean by "new structure"? I'm also not sure why this import statement affects how the analysis toolbox fits.